SurveyTests/Domain/Use Case/Survey/SubmitSurveyUseCaseSpec.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  SubmitSurveyUseCaseSpec.swift
+//  Survey
+//
+//  Created by Khanh on 27/11/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import Combine
+import Nimble
+import Quick
+@testable import Survey
+
+final class SubmitSurveyUseCaseSpec: QuickSpec {
+
+    override func spec() {
+        var submitSurveyUseCase: SubmitSurveyUseCase!
+        var surveyRepository: SurveyRepositoryMock!
+
+        describe("a SubmitSurveyUseCase") {
+
+            beforeEach {
+                surveyRepository = SurveyRepositoryMock()
+                submitSurveyUseCase = SubmitSurveyUseCase(surveyRepository: surveyRepository)
+            }
+
+            context("when submit is called") {
+
+                beforeEach {
+                    _ = submitSurveyUseCase.submit(
+                        id: "d5de6a8f8f5f1cfe51bc",
+                        questionSubmissions: [
+                            QuestionSubmission(
+                                id: "d3afbcf2b1d60af845dc",
+                                answers: [
+                                    AnswerSubmission(id: "f09f1a680789b636459b")
+                                ]
+                            )
+                        ]
+                    )
+                }
+
+                it("returns correct response") {
+                    expect(surveyRepository.submitCalled) == true
+                }
```

Comment:
I suggest add another it block to check the submit arguments, id and array.

Reply:
I added 2 it blocks to check the function submit's arguments and unit test for `SurveyRepository`. Thank you for the feedback 😄 .
Fixed at https://github.com/nkhanh44/ic-survey-ios/pull/84/commits/e2c7dad35fea9d58c8f330e76d76e636cd74ad5e.